static_src/stores/service_instance_store.js

```diff
@@ -150,18 +161,27 @@ class ServiceInstanceStore extends BaseStore {
           action.spaceGuid,
           action.servicePlanGuid
         );
+        this._createLoading = true;
+        this.emitChange();
         break;
       }
 
       case serviceActionTypes.SERVICE_INSTANCE_CREATED: {
         cfApi.fetchServiceInstance(action.serviceInstance.guid);
-        this._createInstanceForm = null;
+        this._createLoading = false;
+        this._createdTempNotification = true;
         this.emitChange();
+        setTimeout(() => {
+          this._createInstanceForm = null;
+          this._createdTempNotification = false;
+          this.emitChange();
+        }, CREATED_NOTIFICATION_TIME_MS);
```

Comment:
I was hesitant to move things to actions in this PR before I see more of how you're doing that. Well, was more thinking what our plan should be there; should we move all our cf api calls to actions or move then one at a time.

I was researching this yesterday and I saw an article that said either call the API from your stores or your actions but not both, which made me hesitant to mix and match.